src/main/java/sysc4806/graduateAdmissions/controller/InterestController.java

```diff
@@ -48,97 +48,45 @@ public ResponseEntity getInterestsOfDepartment(@RequestParam() Department depart
         return ResponseEntity.status(HttpStatus.OK).body(repo.findByDepartment(department));
     }
 
-    /**
-     * serves a form to create a new Interest
-     *
-     * @param model the Model to pass values to the thymeleaf template
-     * @return the name of the form to create a new Interest
-     */
-//    @GetMapping("/create")
-//    public String createInterestForm(Model model) {
-//        model.addAttribute("interest", Interest.builder().build());
-//        model.addAttribute("postLocation", "/interest/create");
-//        model.addAttribute("operation", "Create");
-//        return "interest/setInterestFields";
-//    }
-
     /**
      * adds a new Interest to the data base
      *
      * @param interest the Interest to store
-     * @param model the Model to pass values to the thymeleaf template
-     * @return the name of the result page for adding the template
+     * @return a response indicating the success of the operation
      */
-//    @PostMapping("/create")
-//    public String createInterest(@ModelAttribute Interest interest, Model model) {
-//        repo.save(interest);
-//        model.addAttribute("message", "interest successfully added");
-//        return "interest/result";
-//    }
+    @PostMapping("/create")
+    public ResponseEntity createInterest(@RequestBody Interest interest) {
+        repo.save(interest);
+        return ResponseEntity.ok("interest successfully added");
+    }
```

Comment:
These endpoints should probably be named without the /create, /update bits. See screenshot of the stripe api for example. All of these endpoints would just be /interest except for when theres params.
The GET being /interest/ and the others being /interest/something without the ending slash is also inconsistent. Again, though, it's not wrong technically, but it can be simplified. Writing the full path in each @ xxMapping is fine as well. 
![image](https://user-images.githubusercontent.com/32455969/76273260-d6933280-6253-11ea-8224-d995bbf35d41.png)